src/scripts/ibm-openapi-validator.d.ts

```diff
@@ -15,7 +15,7 @@ declare module "ibm-openapi-validator" {
    * @param openApiDoc An object that represents an OpenAPI document.
    * @param defaultMode If set to true, the validator will ignore the .validaterc file and will use the configuration defaults.
    */
-  function validator(openApiDoc: any, defaultMode = false): Promise<ValidatorResults>;
+  function validator(openApiDoc: any, defaultMode?: any): Promise<ValidatorResults>;
```

Comment:
Even if I need to remove this validator at some point (way to strict to be usable 😕), I'm curious about this change!

Reply:
I reverted the change, and nothing red so far, can you revert this?

Reply:
I changed it back, weird that its no longer red 🤔

Only reason I fiddled with it was because it was red 😄